.fixtures.yml

```diff
@@ -120,5 +120,7 @@ fixtures:
     upstart: https://github.com/simp/pupmod-simp-upstart
     useradd: https://github.com/simp/pupmod-simp-useradd
     xinetd: https://github.com/simp/pupmod-simp-xinetd
+    '../acceptance/rsync_shares/simp':
```

Comment:
We should be doing that for all of our fixtures, yes. This is definitely an improvement than a literal copy of the entire simp-rsync repo into this module's fixtures.